assets/scss/_section.scss

```diff
@@ -0,0 +1,37 @@
+/*doc
+---
+title: Section
+---
+Used to group sets of related components
+into discrete sections by adding padding
+around it's children.
+*/
+
+$sectionSpaceMultiple: 2;
+
+@include responsiveBase() {
+	.section {
```

Comment:
we should be using an additive class pattern with the [sq2 "PCV" convention](https://github.com/meetup/sassquatch2/wiki/Code-standards-&-guidelines#pcv-class-name-convention)...

```
// normal
let className = 'section';

// padded
let className = 'section section--padded`
```

This not only helps us reason about base vs. modifier classes, it also keeps specificity down. You no longer have to nest `padded` inside of `section` in this CSS, instead declaring a basic class, `section--padded`.